pkgs/servers/monitoring/prometheus/openldap-exporter.nix

```diff
@@ -0,0 +1,28 @@
+{ buildGoPackage, lib, fetchFromGitHub }:
+
+buildGoPackage rec {
```

Comment:
We could use buildGoModule instead.

Reply:
I don't see a reason to use this fixed-output drv approach unless we have to though.

Reply:
As long as we don't generate a deps.nix it is whatever tbh.